server/shared/src/main/scala-3/org/http4s/server/staticcontent/FileServiceConfigPlatform.scala

```diff
@@ -0,0 +1,31 @@
+package org.http4s
+package server
+package staticcontent
+
+import scala.annotation.nowarn
+import scala.deriving.Mirror
+
+private[staticcontent] trait FileServiceConfigCompanionPlatform extends Mirror.Product {
```

Comment:
I usually call the version splits "Compat" instead of "Platform", which I guess only matters if there's ever a version and platform split together, but $DEITY forbid.  It's private and it's done and it can be ripped out on main so I wouldn't spend time on it.  Just sharing for next time.

Reply:
Ah, thanks, I was wondering about this. Other projects don't distinguish and I have gotten bitten by this before at least once or twice.